crates/logic/src/server/network/login.rs

```diff
@@ -18,20 +17,20 @@ impl FalconServer {
         let player_uuid = Uuid::new_v3(&Uuid::NAMESPACE_DNS, username.as_bytes());
         let username2 = username.clone();
         connection.execute_sync(move |connection| {
-            // TODO: remove ignore
             connection
-                .send_packet(LoginSuccessSpec::new(player_uuid, username2), falcon_send::write_login_success)
-                .ignore();
+                .send_packet(LoginSuccessSpec::new(player_uuid, username2), falcon_send::write_login_success)?;
```

Comment:
For if formatting allows
```suggestion
            connection.send_packet(LoginSuccessSpec::new(player_uuid, username2), falcon_send::write_login_success)?;
```

Reply:
This is weird, handle with care pls